stdlib/Sockets/src/addrinfo.jl

```diff
@@ -46,7 +46,15 @@ end
     getalladdrinfo(host::AbstractString) -> Vector{IPAddr}
 
 Gets all of the IP addresses of the `host`.
-Uses the operating system's underlying getaddrinfo implementation, which may do a DNS lookup.
+Uses the operating system's underlying `getaddrinfo` implementation, which may do a DNS lookup.
+
+# Example
+```jldoctest
+julia> getalladdrinfo("google.com")
+2-element Array{IPAddr,1}:
+ ip"172.217.6.174"
```

Comment:
google has a lot of IP addresses. `dig` / `host` / `nslookup` seem to bounce between at least 2 or 3 for me over a a short time window.  Perhaps a subdomain (like `ns1.google.com`) or a smaller site / one that we control might be a reliable enough choice?
```
julia> getalladdrinfo("julialang.org")
8-element Array{Sockets.IPAddr,1}:
 ip"151.101.194.49"    
 ip"151.101.130.49"    
 ip"151.101.66.49"     
 ip"151.101.2.49"      
 ip"2a04:4e42::561"    
 ip"2a04:4e42:400::561"
 ip"2a04:4e42:600::561"
 ip"2a04:4e42:200::561"
```